experiments/main_agentic/aggregate.py

```
@@ -0,0 +1,656 @@
+    mode = raw.get("mode")
+    if not isinstance(mode, str) or not mode:
+        raise SystemExit(f"Family config must define a non-empty mode: {config_path}")
+    return mode
+
+
+def _read_run_json(path: Path) -> dict[str, Any] | None:
+    if not path.exists():
+        return None
+    try:
+        data = json.loads(path.read_text(encoding="utf-8"))
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)